modules/fff/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 Google LLC
```

Comment:
I think it's better to put this file under `zephyr/` in your repo `yperess/fff` along with `module.yml`

Reply:
You can see my reasoning here https://android-review.googlesource.com/c/platform/system/chre/+/1833040. It'll be the same as CHRE. I don't want the module maintainers to have to also worry about their downstream clients. We're point to a specific SHA and it is _our_ responsibility when we update the SHA to also make sure that things work.